section1/student.rb

```diff
@@ -0,0 +1,11 @@
+class Student
```

Comment:
Is this the practice from Section 4?

I believe yes, it corresponds with: http://tutorials.jumpstartlab.com/projects/ruby_in_100_minutes.html#11.-objects,-attributes,-and-methods